backend/src/Edna/Web/Server.hs

```diff
@@ -73,6 +74,7 @@ ednaToHandler ctx action =
   `catch` throwPurely @UploadError
   `catch` throwPurely @UploadApiError
   `catch` throwPurely @LibraryError
+  `catch` throwPurely @DashboardError
```

Comment:
Why do we need to list all errors here?

Reply:
Because if you throw an exception, Servant doesn't know how to handle it and treats it as an internal server error, returning the corresponding HTTP code. We want to map all these errors to something different from 500, so we need to catch them and let Servant know what to do with them. And all errors not listed here will be treated as 500.

Reply:
I think I know the solution how to not write it here but will check later